js/NewsChannel.js

```diff
@@ -0,0 +1,128 @@
+var bodyDiv;
+var rightPannel, leftPannel;
+const newsChannels =["All","Channel 1","Channel 2","Channel 3","Channel 4","Channel 5",
+"Channel 6", "Channel 7", "Channel 8", "Channel 9","Channel 10"];
+function GetChannelsInfo(){
```

Comment:
Use camel case for all names in javascript.

Reply:
Updated all function names to camel case